src/scss/webmapviewer-bootstrap-theme.scss

```diff
@@ -65,3 +65,9 @@ $nav-tabs-link-active-bg: $primary;
 .w-13 {
     width: 13%;
 }
+
+@media print {
+    .no-print {
+        display: none !important;
+    }
+}
```

Comment:
This is already present in main.scss which has been added for the tooltip print.